src/core/qgscoordinatereferencesystem.cpp

```diff
@@ -2707,8 +2707,8 @@ int QgsCoordinateReferenceSystem::syncDatabase()
 
   PROJ_STRING_LIST authorities = proj_get_authorities_from_database( pjContext );
 
-  int nextSrsId = 60000;
-  int nextSrId = 520000000;
+  int nextSrsId = 62440;
```

Comment:
@nyalldawson , is there a specific reason you bumped this to 62440?

On my Ubuntu machine (PRJO 6.3.1), I still get SRSID collision, and thus a bunch of SQL errors.

E.g.:

```
Could not execute: INSERT INTO tbl_srs(srs_id, description,projection_acronym,ellipsoid_acronym,parameters,srid,auth_name,auth_id,is_geo,deprecated) VALUES (62440, 'Camacupa 2015','longlat','','+proj=longlat +a=6378249.145 +rf=293.465 +towgs84=-93.799,-132.737,-219.073,-1.844,0.648,-6.37,-0.169 +no_defs',520002440,'EPSG','8694',1,0) [UNIQUE constraint failed: tbl_srs.srs_id/UNIQUE constraint failed: tbl_srs.srs_id]
```

Reply:
Try deleting your local srs6.db and letting it be re-created